app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php

```diff
@@ -98,6 +103,7 @@ public function __construct(
         \Magento\Framework\Stdlib\ArrayUtils $arrayUtils,
         \Magento\Framework\Locale\ResolverInterface $localeResolver,
         \Magento\Framework\App\Helper\AbstractHelper $helperData,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
```

Comment:
Adding a new required parameter to the constructor may break compatibility for any client code that already extends it.
To try and avoid it please refactor it in the backward compatible way. More information is available in the [devdocs](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html#adding-a-constructor-parameter).